Makefile

```diff
@@ -1,10 +1,10 @@
-STACK_VERSION ?= v0.2.3
-STACK_IMG ?= crossplane/stack-gcp-sample:$(STACK_VERSION)
+PACKAGE_VERSION ?= local
```

Comment:
does this env var also get declared/initialized on jenkins? if so, we'll need to update it there too

Reply:
yes, it does look like the 2 need to be in agreement: https://github.com/crossplane/cicd/pull/8#discussion_r416188662

Reply:
Updated in `cicd` :+1: